projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-contact-form/components/ai-assistant-popover/index.tsx

```diff
@@ -11,12 +11,29 @@ import { __ } from '@wordpress/i18n';
 import { PROMPT_TYPE_JETPACK_FORM_CUSTOM_PROMPT, getPrompt } from '../../../../lib/prompt';
 import { AiAssistantUiContext } from '../../ui-handler/context';
 import './style.scss';
+/*
+ * Types
+ */
+import type { UseAiContextOptions } from '@automattic/jetpack-ai-client';
+import type React from 'react';
 
-export const AiAssistantPopover = () => {
+/**
+ * useAiContext hook to provide access to
+ * the AI Assistant data (from context),
+ * and to subscribe to the request events (onDone, onSuggestion).
+ *
+ * @param {UseAiContextOptions} options - the hook options.
+ * @returns {React.Component}          the AI Assistant data context.
+ */
+export const AiAssistantPopover = ( { onSuggestion, onDone, askQuestionOptions } ) => {
 	const { toggle, isVisible, popoverProps, inputValue, setInputValue } =
 		useContext( AiAssistantUiContext );
 
-	const { requestSuggestion, requestingState } = useAiContext();
+	const { requestSuggestion, requestingState } = useAiContext( {
+		onDone,
+		onSuggestion,
+		askQuestionOptions,
+	} );
```

Comment:
```suggestion
	const { requestSuggestion, requestingState } = useAiContext();
```

because of [this](https://github.com/Automattic/jetpack/pull/32209/files#r1281439708)